packages/core/android/src/main/kotlin/com/datadog/reactnative/DdSdk.kt

```diff
@@ -112,7 +112,7 @@ class DdSdk(
      * @param kind Error kind.
      */
     @ReactMethod
-    fun telemetryError(message: String, stack: String?, kind: String?, promise: Promise) {
+    fun telemetryError(message: String, stack: String, kind: String, promise: Promise) {
```

Comment:
we cannot log any error without a stack? why?

Reply:
React Native does not allow optional parameters for iOS methods of native modules.
For now I don't see a case where I won't pass a stack or a kind, if that's the case I'll see what makes more sense, to have another declaration with a different signature or `''` as value.